back/app/serializers/web_api/v1/insights/phase_insights_serializer.rb

```diff
@@ -6,4 +6,8 @@ class WebApi::V1::Insights::PhaseInsightsSerializer < WebApi::V1::BaseSerializer
   attribute :demographics do |_phase, params|
     params[:demographics]
   end
+
+  attribute :participants_and_visitors_chart_data do |_phase, params|
+    params[:participants_and_visitors_chart_data]
+  end
```

Comment:
Happy either way (esp if we name like this elsewhere) but wonder if `participants_and_visitors_time_series` might be better?